keyboards/mzmkb/slimdash/rev1/keyboard.json

```
@@ -0,0 +1,251 @@
+            {"matrix": [9, 5], "x": 134, "y": 51, "flags": 4},
+            {"matrix": [9, 4], "x": 149, "y": 51, "flags": 4},
+            {"matrix": [9, 3], "x": 179, "y": 51, "flags": 4},
+            {"matrix": [9, 2], "x": 194, "y": 51, "flags": 4},
+            {"matrix": [9, 1], "x": 209, "y": 51, "flags": 4},
+            {"matrix": [9, 0], "x": 224, "y": 51, "flags": 4},
+            {"matrix": [8, 0], "x": 224, "y": 38, "flags": 4},
+            {"matrix": [8, 1], "x": 209, "y": 38, "flags": 4},
+            {"matrix": [8, 2], "x": 194, "y": 38, "flags": 4},
+            {"matrix": [8, 3], "x": 179, "y": 38, "flags": 4},
+            {"matrix": [8, 4], "x": 164, "y": 38, "flags": 4},
+            {"matrix": [8, 5], "x": 149, "y": 38, "flags": 4},
+            {"matrix": [7, 6], "x": 134, "y": 26, "flags": 4},
+            {"matrix": [6, 6], "x": 134, "y": 13, "flags": 4},
+            {"matrix": [7, 5], "x": 149, "y": 26, "flags": 4},
+            {"matrix": [7, 4], "x": 164, "y": 26, "flags": 4},
+            {"matrix": [7, 3], "x": 179, "y": 26, "flags": 4},
+            {"matrix": [7, 2], "x": 194, "y": 26, "flags": 4},
+            {"matrix": [7, 1], "x": 209, "y": 26, "flags": 4},
+            {"matrix": [7, 0], "x": 224, "y": 26, "flags": 4},
+            {"matrix": [6, 0], "x": 224, "y": 13, "flags": 4},
+            {"matrix": [6, 1], "x": 209, "y": 13, "flags": 4},
+            {"matrix": [6, 2], "x": 194, "y": 13, "flags": 4},
+            {"matrix": [6, 3], "x": 179, "y": 13, "flags": 4},
+            {"matrix": [6, 4], "x": 164, "y": 13, "flags": 4},
+            {"matrix": [6, 5], "x": 149, "y": 13, "flags": 4},
+            {"matrix": [5, 6], "x": 134, "y": 0, "flags": 4},
+            {"matrix": [5, 5], "x": 149, "y": 0, "flags": 4},
+            {"matrix": [5, 4], "x": 164, "y": 0, "flags": 4},
+            {"matrix": [5, 3], "x": 179, "y": 0, "flags": 4},
+            {"matrix": [5, 2], "x": 194, "y": 0, "flags": 4},
+            {"matrix": [5, 1], "x": 209, "y": 0, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 0, "flags": 4}
+        ],
+        "max_brightness": 128,
+        "sleep": true,
+        "split_count": [35, 35]
+    },
+    "split": {
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "cols": ["GP0", "GP1", "GP3", "GP5", "GP8", "GP7", "GP9"],
+                "rows": ["GP2", "GP4", "GP6", "GP11", "GP10"]
+            }
+        },
+        "serial": {
+            "driver": "vendor"
+        },
+        "transport": {
+            "protocol": "serial",
+            "sync": {
+                "matrix_state": true
+            }
+        },
+        "usb_detect": {
+            "enabled": true
+        }
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7344",
+        "vid": "0x4D5A"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Since this looks to use a 5x14 ortho layout
```suggestion
    "community_layouts": [ "ortho_5x14" ],
    "layouts": {
        "LAYOUT_ortho_5x14": {
```